Partial_Castor_edits.py

```diff
@@ -0,0 +1,53 @@
+# Capture .ser files with single.png for "ser_time" seconds at 1000x1000 resolution
+#
+# Uses user settings for exposure and gain
+# Forces resolution, binning, colour space
+# Returns to users resolution
+#####################################################################################
+
+import time
+
+ser_time = 5 #.ser collection duration in seconds at users exposure
+ 
+str_date = time.strftime("%d%b%Y",time.localtime())
+path = r'C:\DEB\\'+str_date+r'\Partial' # path for main capture folder
+
+str_time = time.strftime("%H%M%S",time.gmtime())
+
+### Initial Setup ####################################################################
+SharpCap.SelectedCamera.LiveView = False
+SharpCap.Settings.UseSubFolders = False
+SharpCap.Settings.CaptureFolder = path
+SharpCap.TargetName = 'partial_'+str_time

```

Comment:
This is kind of nit-picking but "standard" python formatting whitespace guidelines are here: https://peps.python.org/pep-0008/#whitespace-in-expressions-and-statements which would have spaces around the '+' sign.